hexkit/providers/akafka/provider.py

```diff
@@ -69,6 +70,16 @@ class KafkaConfig(BaseSettings):
         example=["localhost:9092"],
         description="A list of connection strings to connect to Kafka bootstrap servers.",
     )
+    security_protocol: str = Field(
+        "PLAINTEXT",
+        description="Protocol used to communicate with brokers. "
+        + "Valid values are: PLAINTEXT, SSL."
+    )
+    ssl_context: ssl.SSLContext = Field(
```

Comment:
This will be hard to configure. Maybe use the arguments of `create_ssl_context` as configuration options instead.